content/id/docs/concepts/overview/object-management-kubectl/imperative-config.md

```diff
@@ -0,0 +1,116 @@
+
+1. Kamu membuat sebuah objek dari sebuah file konfigurasi.
+1. Sebuah sumber lain memperbarui objek tersebut dengan mengubah beberapa *field*.
+1. Kamu memperbarui objek tersebut dengan `kubectl replace` dari file konfigurasi. Perubahan yang dibuat dari sumber lain pada langkah nomor 2 di atas akan hilang.
+
+Jika kamu perlu mendukung beberapa *writer* untuk objek yang sama, kamu bisa menggunakan `kubectl apply` untuk mengelola objek tersebut.
+
+## Membuat dan mengedit objek dari URL tanpa menyimpan konfigurasinya
+
+Misalkan kamu memiliki URL dari sebuah file konfigurasi objek. Kamu bisa menggunakan `kubectl create --edit` untuk membuat perubahan pada konfigurasi sebelum objek tersebut dibuat. Langkah ini terutama berguna untuk mengikuti tutorial atau untuk pekerjaan-pekerjaan yang menggunakan sebuah file konfigurasi di URL terentu yang perlu dimodifikasi.
+
+```sh
+kubectl create -f <url> --edit
+```
+
+## Migrasi dari perintah imperatif ke konfigurasi objek imperatif
+
+Migrsasi dari perintah imperatif ke konfigurasi objek imperatif melibatkan beberapa langkah manual.
+
+1. Ekspor objek *live* ke sebuah file konfigurasi objek lokal:
+```sh
+kubectl get <kind>/<name> -o yaml --export > <kind>_<name>.yaml
+```
+
+1. Hapus secara manual *field* status dari file konfigurasi objek.
+
+1. Untuk pengelolaan objek selanjutnya, gunakan perintah `replace` secara eksklusif.
+```sh
+kubectl replace -f <kind>_<name>.yaml
+```
+
+## Mendefinisikan *controller selectors* dan label PodTemplate
+
+{{< warning >}}
+Memperbarui *selectors* pada *controllers* sangat tidak disarankan.
+{{< /warning >}}
+
+Pendekatan yang direkomendasikan adalah mendefinisikan sebuah label PodTemplate tunggal dan *immutable* yang hanya digunakan oleh *controller selector* tersebut, tanpa makna semantik lainnya.
+
+Contoh label:
+
+```yaml
+selector:
+  matchLabels:
+      controller-selector: "extensions/v1beta1/deployment/nginx"
+template:
+  metadata:
+    labels:
+      controller-selector: "extensions/v1beta1/deployment/nginx"
+```
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+- [Pengelolaan Objek Kubernetes Menggunakan Perintah Imperatif](/docs/concepts/overview/object-management-kubectl/imperative-command/)
+- [Pengelolaan Objek Kubernetes secara Deklaratif dengan Menggunakan File Konfigurasi](/docs/concepts/overview/object-management-kubectl/declarative-config/)
+- [Kubectl Command Reference](/docs/reference/generated/kubectl/kubectl/)
```

Comment:
“Kubectl Command Reference” is in English; is that intentional?

Reply:
hi @sftim, it's unintentional, I missed translating that part. I have updated it in my new commit.

Reply:
Thanks for clarifying & fixing!